src/test/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/service/SDRSServiceIntTest.kt

```diff
@@ -96,7 +96,7 @@ class SDRSServiceIntTest : IntegrationTestBase() {
       assertThat(it.loadType).isEqualTo(FULL_LOAD)
     }
 
-    assertThat(offenceScheduleParts.size).isEqualTo(4)
+    assertThat(offenceScheduleParts.size).isEqualTo(1)
```

Comment:
Again very minor. Can you not just use `assertThat(offenceScheduleParts).hasSize(1)` ?

Reply:
good spot again fixed